src/routes/auth.ts

```diff
@@ -0,0 +1,29 @@
+import { RequestHandler } from 'express';
+
+import { resinApi } from '../platform';
+import { handleHttpErrors } from '../platform/errors';
+
+export const getUserPublicKeys: RequestHandler = (req, res) => {
+	const { username } = req.params;
+
+	if (username == null) {
+		return res.send(400);
+	}
+
+	return resinApi
+		.get({
+			resource: 'user__has__public_key',
+			options: {
+				$select: ['user', 'public_key'],
```

Comment:
I can't see `user` being used in the response at all
```suggestion
				$select: 'public_key',
```